total-rewards.txt

```diff
@@ -1 +1 @@
-11573
\ No newline at end of file
+2787
```

Comment:
I just realized that if we remove the assigned tasks, then it messes with the stats. How can we preserve the stats for marketing purposes? It would be great to write a JSON object with the following:
```typescript
type Rewards = {
  notAssigned: number;
  assigned: number;
  completed: number;
  total: number;
}
```

Also potentially useful stats:
```typescript
type Tasks {
  notAssigned: number;
  assigned: number;
  completed: number;
  total: number;
}
```

Reply:
Why do we need to keep track of total rewards in a separate file? All of the tasks/rewards data can be fetched via github API in a single script.

Reply:
Caching seems appropriate vs paginating tens of pages of results for our index at ubq.fi

Reply:
> keeping some json file in each repository.

No we just need to store it in the `devpool-directory`. Its intended to be system wide statistics, which the cron job already has to process as it updates the mirrored issues states.